src/document/DocumentAnnotator.ts

```diff
@@ -64,6 +60,20 @@ export default class DocumentAnnotator extends BaseAnnotator {
         // Lazily instantiate managers as pages are added or re-rendered
         if (managers.size === 0) {
             if (this.isFeatureEnabled('highlightText')) {
+                const textLayer = pageEl.querySelector('.textLayer') as HTMLElement;
+
+                if (textLayer) {
+                    managers.add(
+                        new HighlightCreatorManager({
+                            getSelection,
+                            location: pageNumber,
```

Comment:
Is `location` used anywhere in `HighlightCreatorManager`?

Reply:
No it's not, I can remove it